### PR TITLE
feat(sig-services): add support process requirement

### LINKED ISSUE #208: [SIG Services][Guideline] Support process
- Support process, how and who to reach out to in all possible cases (outage, feature request, bug etc..)
- Mention SLAs
- How to reach out to the owner and expect response in SLA-agreed time
- Defines a contact point (repo issues, email…)
- Prepared to receive confidential sensitive information and have prepared means for that - as simple as listing a gpg pub key so user can encrypt their data
- Security support process and vulnerability? - Special guideline for that
- Transparent so users are not left waiting without being told they are left waiting
- Most common support requests - feature request, bug, outage, questions should have a category and template prepared
- Have a severity scale so SRE/dev can prioritize
- Don’t close support request until the issue resolution is available to end user - PR merge to master is not enough.

### PR DESCRIPTION
**Pull request by SamoKopecky**
Closes #208 

Might be a bit too long, but I can make it shorter. Let me know what you think.

### CODE REVIEW

File: sig-services/service-requirements/support-process.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+An open service should have a support process defined. A support process is a way for users to reach out to the service owner/provider or developers and report any possible issues that might arise from the service e.g. service outage, feature request, or bug report. The support process should be transparent so that other users can see already solved issues and prevent duplicates and also see the progress of the reported issues. Response time to requests/reports shouldn't exceed the agreed time to respond and leave the user waiting for an answer, especially if the support process is also transparent. If the reported issue is a feature request or a bug the issue should not be closed before the change is available to the user in the next version.
+
+If a user needs to send any confidential information that is required for the issue, there should exist a way for the user to ensure confidentiality of the information, for example with some kind of encryption. The support process mustn't include a way to report vulnerabilities and security issues.
+
+
+## Requirements
+
+- Define a contact point for the support process (GitHub issues, email, ...)
```
Comment (Shreyanand) [inline sig-services/service-requirements/support-process.md:14]: Should we recommend a place for defining the support process? maybe the user docs or readme of the Github service.

**Review by Shreyanand — COMMENTED** [review]
Minor comments :)

File: sig-services/service-requirements/support-process.md (hunk 1 above)
Comment (SamoKopecky) [reply sig-services/service-requirements/support-process.md:14]: That's a good idea I'll just add another sub-requirement that the support process should be documented in some place too.

**Comment by SamoKopecky** [comment]
Thanks for the feedback Shrey, PR is ready for review again.

**Review by tumido — COMMENTED** [review]
I think this should pick/link to https://github.com/operate-first/sre/blob/main/sre_maturity.md in some way

**Comment by SamoKopecky** [comment]
> I think this should pick/link to https://github.com/operate-first/sre/blob/main/sre_maturity.md in some way

Added in the latest commit.

**Review by Shreyanand — COMMENTED** [review]
/lgtm

**Pull request closed without merging**